navidrome/docker-compose.yml

```
@@ -0,0 +1,20 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: navidrome_server_1
+      APP_PORT: 4533
+      PROXY_AUTH_ADD: "false"
+  server:
```

Maybe add a blank here to make the readability a bit better, but this is just a very minor nitpick
```suggestion
      PROXY_AUTH_ADD: "false"

  server:
```

---

this change looks good.